src/styles/main.scss

```diff
@@ -1,3 +1,21 @@
-body {
+@import "./blocks/header.scss";
+@import "./blocks/card.scss";
+@import "../utils/variables.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/price.scss";
+@import "./blocks/button.scss";
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  border: none;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
+  gap: 46px 48px;
+  padding-bottom: 72px;
+  padding-top: 50px;
 }
```

Comment:
since you're using grid - please try to implement those styles: [styles](https://github.com/PawelP2001/layout_catalog/blob/develop/grid.md)